hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OmMetadataManagerImpl.java

```diff
@@ -965,23 +964,27 @@ public boolean isBucketEmpty(String volume, String bucket)
 
     // Clear map and set.
     cacheKeyMap.clear();
-    deletedKeySet.clear();
 
     return result;
   }
 
   // TODO: HDDS-2419 - Complete stub below for core logic
   @Override
-  public List<RepeatedOmKeyInfo> listTrash(String volumeName, String bucketName,
-      String startKeyName, String keyPrefix, int maxKeys) throws IOException {
+  public List<RepeatedOmKeyInfo> listTrash(String volumeName,
```

Comment:
It would be better to revert these formatting changes. The "Ozone style" seems to be more like the unchanged version, and these sort of changes can cause conflicts etc later.

Reply:
The check style script locally complained about the width for those lines..

Reply:
How do you run it and what's the output?

Reply:
On my local checkout, checkstyle is happy with these lines - eg line 975 is exactly 80 characters. Can you double check this after reverting these changes?